### PR TITLE
ci_build tweaks to fix error output

### PR DESCRIPTION
**Pull request by wjordan**

### CODE REVIEW

File: aws/ci_build [hunk 1]
```diff
@@ -91,6 +95,7 @@ def main()
     commit_url = "https://github.com/code-dot-org/code-dot-org/commit/#{RakeUtils.git_revision}"
     # Notify the HipChat channels about what happened.
     projects = ARGV.join(' ')
+    projects = 'websites' if projects.empty?
```
Comment (ashercodeorg) [inline aws/ci_build:98]: Can this be `projects = if !ARGV.empty? ? ARGV.join(' ') : 'websites'` or some such?

File: aws/ci_build [hunk 2]
```diff
@@ -104,6 +109,7 @@ def main()
 
       HipChat.log message, color: 'red'
       HipChat.log "/quote #{log}", color: 'gray', message_format: 'text'
+      HipChat.log "/quote #{status}", color: 'gray', message_format: 'text'
```
Comment (ashercodeorg) [inline aws/ci_build:112]: Why are we adding new logging to HipChat? Should there be a TODO to convert this logging to Slack?

Comment (wjordan) [inline aws/ci_build:112]: the `HipChat` class sends messages to both HipChat and Slack in its current implementation. If there's a TODO, it should be bulk-renaming the HipChat class to something more generic like `ChatLog` or something (and out of the scope of this particular fix).

File: aws/ci_build (hunk 1 above)
Comment (wjordan) [inline aws/ci_build:98]: it could, but I think the alternative ends up being roughly just as complex so it's a bit of a wash.

**Pull request merged**